discover/commerce/articles/01-catalog-management/02-products/00-product-types-intro.markdown

```diff
@@ -0,0 +1,111 @@
+    at the top of the screen (and can be reached again at any time by navigating to
+    the catalog and selecting a product). The first tab, *Details*, is populated
+    with your responses from the *Add Product* form.
+
+3.  Enter additional information specific to the product type you selected in
+    step 1.
+
+    **For a Simple Product**, there is no additional information to enter. When
+    you create simple product, the standard product tab bar is displayed,
+    starting with a *Details* tab where you can edit entries from the *Add
+    Product* form.
+
+    **When a Grouped Product** is created, you need to specify the products that
+    it will contain. A grouped product has an extra tab--*Grouped*--in its
+    product tab bar. From that tab, click on the
+    ![Add](../../../images/icon-add.png) button in the upper-right to select
+    any number of products. Once products have been added to the group, they can
+    be edited individually (![options](../../../images/icon-options.png)
+    &rarr; *Edit*) to set a quantity and a priority (priority indicates the
+    order in which items will be displayed--lower numbers come first).
+
+    ![Figure 1: The features that make a grouped product unique can be conveniently found in the *Grouped* tab.](../../../images/grouped-product.png)
+
+    **A Virtual Product** also has an extra tab--*Virtual*--in its product tab
+    bar. The *Virtual* tab contains the following fields which should be filled
+    in if applicable:
+
+**URL/Select File**: If your virtual product is stored at a web address, enter
+the URL here. Alternatively, click *Select* to upload a local file or select one
+that was uploaded previously.
+
+**Activation Status**: Select the stage in the order process at which the
+product becomes available for download.
+
+**Duration**: Enter the number of days after purchase in which customers
+will have access to the file. A value of 0 sets no limit.
+
+**Max Usages**: Set the number of times a customer can download the file.
+A value of 0 sets no limit.
+
+**Sample**: Upload or enter a URL for a sample file that will be
+available to prospective buyers. The *Use Sample* box must be checked
+before customers will see the sample.
+
+**Terms of Use**: Either enter a Terms of Use document directly into the
+integrated editor, or select a document previously uploaded to Web
+Content.
+
+### Add Product Form [](id=add-product-form)
+
+The *Add Product* form contains the following fields:
+
+**Title**: Name your product. This is the only required field.
+
+**Short Description**: Enter an abstract of the product's full description if
+you need to display brief version. Text entered here could be displayed where
+the product appears in thumbnail form or in search results--anywhere space is
+limited.
+
+**Full Description**: This text would typically be displayed to customers on the
+product page. The integrated text editor provides a number of formatting options
```

Comment:
Not sure you need to descirbe the text editor so much, as it appears all over Liferay.